main.css

```diff
@@ -38,8 +50,66 @@ body {
 #timer {
   width: 100%;
   text-align: center;
-  color: #FBA90A;
+  color: #FDDC22;
   font-size: 200%;
   font-family: 'Courier New', Courier, monospace;
   font-weight: bolder;  
+}
+
+.modal {
+  display: block;
+  position: fixed;
+  z-index: 1;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgb(0,0,0);
+  background-color: rgba(0,0,0,0.4);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.modal-content {
+  background-color: #D4DAF7;
+  border: 2px solid #000000;
+  width: 20%;
+  display: flex;
+  flex-wrap: unset;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+#submit {
+  color: #F980F5;
+  float: right;
+  font-size: 2.5vw;
+  font-weight: bold;
+  margin: 5px;
+  background-color: rgb(253, 253, 253);
+}
+
+#submit:focus {
+  color: #9A0794;
+  text-decoration: none;
+  cursor: pointer;
+}
+.radiocontainer {
+  width: 35%; 
+  font-size: 1.4vw;

```

Comment:
Consider using `rem` instead of `vw` as it may cause unexpected behavior.
Text too small to read
<img width="400" alt="Screen Shot 2019-03-25 at 00 30 54" src="https://user-images.githubusercontent.com/6773202/54886848-58e4e400-4e95-11e9-8ac2-bc7244a8bdf3.png">